index.bs

```diff
@@ -242,6 +267,19 @@ orientation sensor's [=latest reading=] would represent 0 (rad) [[SI]] rotation
 
 <img src="images/absolute_orientation_sensor_coordinate_system.png" srcset="images/absolute_orientation_sensor_coordinate_system.svg" style="display: block;margin: auto;" alt="AbsoluteOrientationSensor coordinate system.">
 
+The RelativeOrientationSensor Model {#relativeorientationsensor-model}
+----------------------------------------------------------------------
+
+The {{RelativeOrientationSensor}} class is a subclass of {{OrientationSensor}} which represents the [=Relative
+Orientation Sensor=].
+
+For the relative orientation sensor the value of [=latest reading=]["quaternion"] represents the rotation of a
+device hosting motion sensors in relation to a stationary reference coordinate system.
```

Comment:
+1

Make the "stationary reference coordinate system" a `<dfn>` similarly to Earth’s reference coordinate system in https://w3c.github.io/orientation-sensor/#earths-reference-coordinate-system